arrow/arrow.py

```diff
@@ -1078,8 +1111,10 @@ def humanize(
                     )
 
                 if trunc(abs(delta)) != 1:
-                    granularity += "s"
-                return locale.describe(granularity, delta, only_distance=only_distance)
+                    granularity += "s"  # type: ignore
```

Comment:
I find it odd that mypy marks this as incorrect despite it being in a block that checks if granularity is a string